imaging/imaging.py

```diff
@@ -12,7 +12,7 @@
 CAMERA_WIDTH = 1920
 CAMERA_HEIGHT = 1200
 
-SAVE_PREFIX = "log_record_" + str(int(time.time())) + "_"
+SAVE_PREFIX = "images/log_record_" + str(int(time.time())) + "_"
```

Comment:
Is this successful if the directory doesn't exist?

Reply:
No, but I've added the directory with a placeholder to the repo.